x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/index_input.tsx

```diff
@@ -34,31 +34,35 @@ export const InferenceInputFormIndexControls: FC<Props> = ({ inferrer, data }) =
   } = data;
 
   const runningState = useObservable(inferrer.getRunningState$());
+  const pipeline = useObservable(inferrer.getPipeline$());
```

Comment:
you also need an initial value 
```suggestion
  const pipeline = useObservable(inferrer.getPipeline$(), inferrer.getPipeline());
```

Reply:
Updated in https://github.com/elastic/kibana/pull/145357/commits/d70170fa0e3879cafb6097a71aa4383ca918714c